fuse/eval/metrics/libs/classification.py

```diff
@@ -36,6 +36,7 @@ def auc_roc(
         target: Sequence[Union[np.ndarray, int]],
         sample_weight: Optional[Sequence[Union[np.ndarray, float]]] = None,
         pos_class_index: int = -1,
+        ignore_index: int = None,
```

Comment:
ignore_index: Optional[int] = None,
Can you please add it in the comments as well?

Reply:
done